extensions/parquet/table/src/main/java/io/deephaven/parquet/table/BigDecimalParquetBytesCodec.java

```diff
@@ -27,7 +27,7 @@ public class BigDecimalParquetBytesCodec implements ObjectCodec<BigDecimal> {
      *        precision.
      */
     public BigDecimalParquetBytesCodec(final int precision, final int scale, final int encodedSizeInBytes,
-            final RoundingMode roundingMode) {
+                                       final RoundingMode roundingMode) {
```

Comment:
I'm pretty sure spotless will put this back the way it was.

Reply:
See the second commit in this branch - the first was "transfer changes from dhe", the second was "run spotless"